autoendpoint/src/db/retry.rs

```diff
@@ -48,6 +49,9 @@ pub fn retryable_describe_table_error(
 }
 
 /// Build an exponential retry policy
-pub fn retry_policy() -> RetryPolicy {
-    RetryPolicy::exponential(Duration::from_millis(100))
+pub fn retry_policy(jitter: u64) -> RetryPolicy {
```

Comment:
Alternatively we could switch to `RetryPolicy::default()` which is `Exponential` at 1 second -- because this is what the connection server uses. I wonder if a jitter is even all that necessary with exponential backoff increasing the delay after every retry, but it wouldn't hurt -- though both retry libs also support jitter natively `RetryPolicy::default().with_jitter(true)`